tools/pot/openvino/tools/pot/graph/passes.py

```diff
@@ -832,6 +893,39 @@ def insert_fake_quantize(graph, node, ports=None, names=None):
     return new_fq
 
 
+def insert_output_fake_quantize(graph, node, hw_config=None, ignored_params=None):
```

Comment:
Sorry, I didn't get how we avoid inserting extra FQs in the patterns like Conv+Bias_Add here because following the logic it should be Conv->FQ->Bias_add which is incorrect and the proper graph should be Conv->Bias_add->FQ.